README.md

```diff
@@ -44,6 +44,11 @@ parameter | explanation
 `stealth.user_agent` |  A string representing a user agent that is used by the stealth module as a user agent value. It can be an empty string. If this query parameter does not exist, it is treated as not enabled.
 `stealth.ip` |  A string representing a IP address that is used by the stealth module as a ip address. If this query parameter does not exist, it is treated as not enabled.
 `stealth.dpi` | Can be `true` or `false`.
+`stealth.block_trackers` | Can be `true` or `false`.
+`stealth.disable_wap_push_message_routing_service` | Can be `true` of `false`.
```

Comment:
what stealth options is this parameter for?

Reply:
![Screenshot 2023-10-20 at 10 16 22](https://github.com/AdguardTeam/ReportsWebApp/assets/2804667/36171c35-6fd4-4769-99d4-4095f3b4f1ad)

This is the new Windows tracking settings

Reply:
@slavaleleka [For example this feature on macOS.](https://uploads.adguard.org/Chinaski71n2c.png)